core/server/master/src/test/java/alluxio/master/AlluxioMasterRestServiceHandlerTest.java

```diff
@@ -91,9 +90,14 @@
   private BlockMaster mBlockMaster;
   private MasterRegistry mRegistry;
   private AlluxioMasterRestServiceHandler mHandler;
+
   @Rule
   public TemporaryFolder mTestFolder = new TemporaryFolder();
 
+  @Rule
+  public ConfigurationRule mConfigurationRule = new ConfigurationRule(
+      new ImmutableMap.Builder().put(PropertyKey.MASTER_MOUNT_TABLE_ROOT_UFS, TEST_PATH).build());
```

Comment:
Thanks, I didn't know about that constructor. What do you think of 
```java
  public ConfigurationRule mConfigurationRule = new ConfigurationRule(new HashMap() {{
      put(PropertyKey.MASTER_MOUNT_TABLE_ROOT_UFS, TEST_PATH);
  }});
```
instead? I like it since the key/value are on the same line and it's easy to add new configuration if necessary.